cmd/mirror.go

```diff
@@ -183,6 +188,72 @@ func delComp(repo, id, version string) error {
 	return nil
 }
 
+// the `repo publish` sub command
+func newMirrorPublishCmd(env *meta.Environment) *cobra.Command {
+	privPath := env.Profile().Path("private.json")
+	endpoint := "http://127.0.0.1:8989"
+	goos := runtime.GOOS
+	goarch := runtime.GOARCH
+	desc := ""
+
+	cmd := &cobra.Command{
+		Use:   "publish <comp-name> <version> <tarbal> <entry>",
+		Short: "Publish a component",
+		Long:  "Publish a component to the repository",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 4 {
+				return cmd.Help()
+			}
+			log.Debugf("Publish component %s", args[0])
```

Comment:
Please use `fmt.Println`. The logger should be used in tiup-server, instead of client.